src/docs/Foundations/Typography/BodyStyles/index.mdx

```diff
@@ -1,27 +1,46 @@
 ---
 title: 'Body styles'
 order: 4
+description: 'A body is a paragraph or a multiline text of a pattern or a page. It must provide basic typographic styles like font-size, line-height and font-weight.'
```

Comment:
why "of a pattern or a page" could we not just say "the body style apply to paragraphs or multiline texts", and then not "it must provide" but just "it provide".